### PR TITLE
Read bower config from correct directory

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by justinfagnani** [comment]
@rictic this one will be a little easier to review commit-by-commit

File: bin/polyserve [hunk 1]
```diff
@@ -9,8 +9,10 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
```
Comment (rictic) [inline bin/polyserve]: Can replace all uses of `let` with `const` in this file

File: src/bower_config.js [hunk 2]
```diff
@@ -8,16 +8,17 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
-var path = require('path');
-var fs = require('fs');
+'use strict';
+
+let path = require('path');
+let fs = require('fs');
```
Comment (rictic) [inline src/bower_config.js]: const requires seems nice for legibility

File: src/make_app.js [hunk 3]
```diff
@@ -8,34 +8,40 @@
...
-var send = require('send');
-var bowerConfig = require('./bower_config');
+'use strict';
+
+let express = require('express');
+let fs = require('fs');
+let path = require('path');
+let parseUrl = require('url').parse;
+let send = require('send');
+let bowerConfig = require('./bower_config');
```
Comment (rictic) [inline src/make_app.js]: Ditto

File: src/bower_config.js (hunk 2 above)
Comment (justinfagnani) [inline src/bower_config.js]: done

File: bin/polyserve (hunk 1 above)
Comment (justinfagnani) [inline bin/polyserve]: done

File: src/make_app.js (hunk 3 above)
Comment (justinfagnani) [inline src/make_app.js]: done

File: src/start_server.js [hunk 4]
```diff
@@ -41,36 +52,48 @@ function startWithPort(options) {
...
+    let url = baseUrl + (options.page === true ? 'index.html' : options.page);
     if (Array.isArray(options.browser)) {
-      for (var i = 0; i < options.browser.length; i++)
+      for (let i = 0; i < options.browser.length; i++)
         opn(url, options.browser[i]);
     }
     else {
       opn(url, options.browser);
     }
   }
```
Comment (rictic) [inline src/start_server.js]: Consider waiting until after the server has actually started before telling the user or starting any browsers.

**Comment by rictic** [comment]
LGTM with the above comments.

File: src/start_server.js (hunk 4 above)
Comment (justinfagnani) [inline src/start_server.js]: It should already be doing that. `app.listen` takes a callback that's called after the server starts.http://expressjs.com/4x/api.html#app.listen which references https://nodejs.org/docs/latest/api/net.html#net_server_listen_port_hostname_backlog_callback

Comment (rictic) [inline src/start_server.js]: Yeah, but here we're calling app.listen and then immediately doing a console.log() and starting browsers, right?

I'm suggesting that we should put that logic inside of `serverStartedPromise.then(() => { ... })`

Comment (justinfagnani) [inline src/start_server.js]: Ah yes, sorry didn't parse your whole comment... good point.

**Pull request merged**